src/core_modules/capture-core/cleanUp/cleanUp.js

```diff
@@ -11,7 +11,7 @@ function getActionToCleanUpInLoadingList(state: ReduxState) {
 
 export default function cleanUp(store: ReduxStore) {
     const cleanUpActions = [
-        getActionToCleanUpInLoadingList(store.getState()),
```

Comment:
updating `redux-obervables` came with the redundancy of the function `getState` now we can access the store in the `epics` by using the `store.value`